src/simularium/VisData.ts

```diff
@@ -357,6 +358,7 @@ class VisData {
         }
         this.frameCache = [];
         this.frameDataCache = [];
+        this.firstFrameTime = -1; // Assumes negative timestamps don't exist
```

Comment:
you could use null or undefined as the uninitialized value to avoid this assumption

Reply:
Oh good point, for some reason I thought it would be difficult to deal with TS that way, but it was no problem, fixed!